crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__RUF052_RUF052.py.snap

```diff
@@ -0,0 +1,55 @@
+---
+source: crates/ruff_linter/src/rules/ruff/mod.rs
+snapshot_kind: text
+---
+RUF052.py:52:5: RUF052 Local variable `_invalid_var` is marked as unused but is used
+   |
+51 | def fun(_valid_arg):
+52 |     _invalid_var = _valid_arg  # [RUF052]
```

Comment:
Can you tell me a bit more about the reasoning for excluding function parameters from checking? I would have expected that they are handled the same. Or is it to avoid breaking call sites if calls use named arguments?

Reply:
You are right, parameters should be addressed in the same way.